site/gulpfile.js

```diff
@@ -1,15 +1,18 @@
 var gulp = require('gulp');
 var elixir = require('laravel-elixir');
+var argv = require('yargs').argv;
 
 elixir.config.assetsPath = 'source/_assets';
 elixir.config.publicPath = 'source';
 
 elixir(function(mix) {
+	var env = argv.e || argv.env || 'local';
+
     mix.sass('main.scss')
-        .exec('jigsaw build', ['./source/*', './source/**/*', '!./source/_assets/**/*'])
+        .exec('jigsaw build --env ' + env, ['./source/**/*', '!./source/_assets/**/*'])
```

Comment:
Yes it should in CLI applications, you can use an equals sign or a space. But it can be ambiguous if the parameter is an argument of the option, or another argument. I've changed it to be an equals to remove the ambiguity.